docs/application/toc_all.md

```diff
@@ -144,9 +144,11 @@
 ##### [Overview](/application/dotnet/guides/multimedia/media-vision.md)
 ##### [Barcode Detection and Generation](/application/dotnet/guides/multimedia/image-barcode.md)
 ##### [Deep Learning Based Face Recognition](/application/dotnet/guides/multimedia/face-recognition.md)
-##### [Face Detection, Recognition, and Tracking](/application/dotnet/guides/multimedia/face-detection.md)
-##### [Image Recognition and Tracking](/application/dotnet/guides/multimedia/image-recognition.md)
-##### [Pose Detection](/application/dotnet/guides/multimedia/pose-detection.md)
+##### [Deep Learning Based Face Detection](/application/dotnet/guides/multimedia/face-detection.md)
+##### [Deep Learning Based Object Detection](/application/dotnet/guides/multimedia/object-detection.md)
+##### [Deep Learning Based Image Classification](/application/dotnet/guides/multimedia/image-classification.md)
+##### [Deep Learning Based Facial Landmark](/application/dotnet/guides/multimedia/facial-landmark.md)
+##### [Deep Learning Based Pose Landmark](/application/dotnet/guides/multimedia/pose-landmark.md)
```

Comment:
@daeinki Sorry, I just realised that the changes in the `toc_all.md` file have been made in the dotnet part of the Visual Recognition, not the native part, which starts from Line 951. Please remove the changes here and make the changes in the native part.